wallet-test/src/test/scala/org/bitcoins/wallet/models/DLCDAOTest.scala

```diff
@@ -0,0 +1,199 @@
+    for {
+      _ <- {
+        val dlcDb = DLCDb(
+          eventId = accept.eventId,
+          isInitiator = true,
+          account = HDAccount.fromPath(BIP32Path.fromString("m/84'/0'/0'")).get,
+          keyIndex = 0,
+          winSigOpt = None,
+          loseSigOpt = None,
+          refundSigOpt = None,
+          oracleSigOpt = None
+        )
+        dlcDAO.create(dlcDb)
+      }
+      _ <- {
+        val acceptDb = DLCAcceptDb.fromDLCAccept(accept)
+        acceptDAO.create(acceptDb)
+      }
+
+      readAccept <- acceptDAO.read(accept.eventId)
+    } yield {
+      assert(
+        readAccept.get
+          .toDLCAccept(accept.fundingInputs)
+          .toJson == accept.toJson)
+    }
+  }
+
+  it should "correctly insert funding inputs into the database" in { daos =>
+    val inputsDAO = daos.dlcInputsDAO
+    val dlcDAO = daos.dlcDAO
+
+    val eventId = DLCWalletTestUtil.sampleDLCEventId
+
+    val input = DLCFundingInputDb(
+      eventId = eventId,
+      isInitiator = true,
+      outPoint = TransactionOutPoint(testBlockHash, UInt32.zero),
+      output = TransactionOutput(Satoshis.one, EmptyScriptPubKey),
+      sigs = Vector(DLCWalletTestUtil.dummyPartialSig)
+    )
+
+    for {
+      _ <- {
+        val dlcDb = DLCDb(
+          eventId = eventId,
+          isInitiator = false,
+          account = HDAccount.fromPath(BIP32Path.fromString("m/84'/0'/0'")).get,
+          keyIndex = 0,
+          winSigOpt = None,
+          loseSigOpt = None,
+          refundSigOpt = None,
+          oracleSigOpt = None
+        )
+        dlcDAO.create(dlcDb)
+      }
+
+      _ <- inputsDAO.create(input)
+      readInput <- inputsDAO.findByEventId(eventId)
+    } yield assert(readInput.head.toString == input.toString)
+  }
+
+  it should "correctly find funding inputs by eventId and isInitiator" in {
+    daos =>
+      val inputsDAO = daos.dlcInputsDAO
+      val dlcDAO = daos.dlcDAO
+
+      val eventId = DLCWalletTestUtil.sampleDLCEventId
+
+      val inputs = Vector(
+        DLCFundingInputDb(
+          eventId = eventId,
+          isInitiator = true,
+          outPoint = TransactionOutPoint(testBlockHash, UInt32.zero),
+          output = TransactionOutput(Satoshis.one, EmptyScriptPubKey),
+          sigs = Vector(DLCWalletTestUtil.dummyPartialSig)
+        ),
+        DLCFundingInputDb(
+          eventId = eventId,
+          isInitiator = false,
+          outPoint = TransactionOutPoint(testBlockHash, UInt32.one),
+          output = TransactionOutput(Satoshis.one, EmptyScriptPubKey),
+          sigs = Vector(DLCWalletTestUtil.dummyPartialSig)
+        ),
+        DLCFundingInputDb(
+          eventId = eventId,
+          isInitiator = true,
+          outPoint = TransactionOutPoint(testBlockHash, UInt32(3)),
+          output = TransactionOutput(Satoshis.one, EmptyScriptPubKey),
+          sigs = Vector(DLCWalletTestUtil.dummyPartialSig)
+        )
+      )
+
+      for {
+        _ <- {
+          val dlcDb = DLCDb(
```

Comment:
Since the `DLCDb`s in this file are the same across tests, it is probably worth throwing this val up in `DLCWalletTestUtil` with the rest of the vals. (Same might apply for offer/accept/funding inputs)